src/types/subscription.ts

```diff
@@ -160,25 +160,41 @@ export interface ParticipationUpdates {
  * * Add emitted ARC-28 events via `arc28Events`
```

Comment:
The `upgradeDelay` a few lines above - I know it's existing, but maybe we can review the `number | bigint`?